src/components/input-number/input-number.e2e.ts

```diff
@@ -1407,6 +1407,59 @@ describe("calcite-input-number", () => {
     });
   });
 
+  it("allows disabling slotted action", async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      `<calcite-input-number><calcite-button slot="action" disabled>Action</calcite-button></calcite-input-number>`
+    );
+
+    const input = await page.find("calcite-input-number");
+    const button = await page.find("calcite-button");
+
+    await input.callMethod("setFocus");
+    await typeNumberValue(page, "1");
+    await page.waitForChanges();
+    expect(await input.getProperty("value")).toBe("1");
+    expect(await button.getProperty("disabled")).toBeTruthy();
+    expect(await input.getProperty("disabled")).toBeFalsy();
```

Comment:
nitpick: any reason why `disabled` values won't always be either `true` or `false` in order to use a more explicit assertion like `toBe(true|false)`?

Reply:
If whatever the user puts in the slot doesn't reflect the `disabled` attribute it could be `""` or `null`

Reply:
Actually I think this one can be changed since I'm explicitly getting the property. I'll double check